src/FTPBase.Handler.ts

```diff
@@ -36,7 +36,12 @@ export abstract class FTPBaseHandler implements ICommandHandler {
             await this.processFTP(additionalParameters);
         } catch (e) {
             this.log.error("Error encountered in FTP command:\n%s", require("util").inspect(e));
+            if (e.message.indexOf("PASS command failed") !== -1) {
```

Comment:
these code changes make sense for #85 but I don't see any code changes that address #81.  listing a loadlib data set I still get:
```
Command Error:
Unrecognized file list header:  Name      Size     TTR   Alias-of AC --------- Attributes --------- Amode Rmode
```

Reply:
Thank you @dkelosky, to support listing loadlib dataset members, the main code change is in zos-node-accessor, and I just update the zos-node-accesor’s version in package.json. Now it is 1.0.9 and you may need run npm install again to update the modules.

Reply:
Shouldn't we make the regular dependencies static?
Meaning no `^` on `package.json#dependencies`
https://github.com/zowe/zowe-cli-ftp-plugin/blob/d7f541a493c24dfd1ea6bf2fbb1acee3608e5224/package.json#L36-L38

Reply:
Agree to use static version number in dependencies, to avoid the potential npm subtitution attacks.

Reply:
Thanks @zFernand0 @std4lqi ! I will update it. Thank you very much!